lib/internal/modules/esm/loader.js

```diff
@@ -431,6 +373,7 @@ class ESMLoader {
     const moduleProvider = async (url, isMain) => {
       const {
         format: finalFormat,
+        responseURL,
```

Comment:
Sure we can go with `responseUrl` here too. That is the term explicitly used in the fetch and WhatWG specs so I don't think we should use another name for this.

Reply:
One thing worth noting here is we currently use the term `parentURL` in the loader API, so that having `parentURL` and `reponseUrl` may be an inconsistency.

Reply:
I like your idea here, but given `parentURL` already breaks this pattern, consistency should likely be the goal.

Also note that WhatWG explicitly says not to use `URL` in APIs, so an API that wanted to mean `URL` would be going against WhatWG guidance...

Reply:
I've been wanting to change parentURL too for the same reason 😅

>That is the term explicitly used in the fetch and WhatWG specs

Gah, actually, now that you mention it, I think I asked the same question to Bradley for his original implementation and he mentioned that too 🤦‍♂️

Reply:
So should it be `parentUrl`? Let's just change it while we're still experimental.

Reply:
Yep!

Reply:
Sure, I've done the renaming for both.

Reply:
can we rename both to "URL"? initialisms should always be consistently cased; `Url` is an incorrect Java-ism

Reply:
I don't have a strong opinion either way, but we do need to come to agreement and it would be nice not to delay this PR further on this kind of point. @Trott wondering if perhaps there is any existing guidance for this one for `URL` v `Url`?

Reply:
> Also note that WhatWG explicitly says not to use `URL` in APIs, so an API that wanted to mean `URL` would be going against WhatWG guidance…

Is there a citation for this? I think should follow whichever standard the `fetch` API uses, since we’re forced to use either `URL` or `Url` per the spec for `fetch`, and so we might as well be consistent with that for here.

Reply:
URL spec guidance is to use capitalized form when in a compound word : https://url.spec.whatwg.org/#url-apis-elsewhere . Note: the recomendation to avoid URLs is about avoiding using the URL class and instead says you should use a safe serialized form, not against using the term itself.

Reply:
All uses of `URL` in web spec functions use the full title case - `createObjectURL`, `URLSearchParams`, `readAsDataURL` etc. Fetch doesn't actually specify any `responseURL` instead it's just `Response.url`. The note about `URL` in APIs is here - https://url.spec.whatwg.org/#url-apis-elsewhere.

Reply:
Ah sorry didn't see the latest comments - ok I've reverted the renaming.